### DISCUSSION TITLE
Request Body Data

### DISCUSSION THREAD

**Discussion by SepehrBazyar** [cirospaciari/socketify.py#142] [Q&A] [answered]
Hello @cirospaciari 
First of all, thank you for this wonderful project.

My question is that the data sent in the request body can be accessed by the `res.get_json()` instead of being read from `req.get_json()`?
Please explain the reason for this implementation? Thanks

**Comment by cirospaciari** [comment, marked as answer]
> Hello @cirospaciari 
> First of all, thank you for this wonderful project.
> 
> My question is that the data sent in the request body can be accessed by the `res.get_json()` instead of being read from `req.get_json()`?
> Please explain the reason for this implementation? Thanks

The reason is the uWebSockets C++ interface does the same, probably will change that in future because it is not very intuitive
